### PR TITLE
CB-3827 YARN E2E fail fix

### PR DESCRIPTION
**Pull request by bergerdenes**
S3 cloudstorage creation in TelemetryTestDto should depend on
cloud provider type

Remove mistakenly committed file (application.yml)

### CODE REVIEW

File: integration-test/application.yml [hunk 1]
```diff
@@ -1,438 +0,0 @@
-spring:
-    freemarker:
-        checkTemplateLocation: false
-
-# The logfiles will be created in this directory, LOG_PATH system property will be set and can be used in logback.xml
-# http://docs.spring.io/spring-boot/docs/current/reference/html/boot-features-logging.html#boot-features-logging-file-output
-logging:
-    path: ${integrationtest.outputdir}
-
-integrationtest:
```
Comment (attilapalfi92) [inline integration-test/application.yml]: These default values are required for e2e tests to work

Comment (attilapalfi92) [reply integration-test/application.yml]: application.yml is part of the integration-test project

Comment (attilapalfi92) [reply integration-test/application.yml]: also necessary for mock tests

Comment (bergerdenes) [reply integration-test/application.yml]: the file itself was committed by mistake. the correct one resides under src/main/resources

**Pull request merged**